### PR TITLE
add attribute filters to ogr source

### PR DESCRIPTION
**Pull request by michaelmattig**

### CODE REVIEW

**Comment by michaelmattig** [comment]
[Filter in der OgrSource](https://trello.com/c/MmWS8ix3/607-filter-in-der-ogrsource)

File: operators/src/source/ogr_source/mod.rs [hunk 1]
```diff
@@ -470,6 +535,74 @@ impl FeaturesProvider<'_> {
...
+                        start = n.start(),
+                        stop = n.end()
+                    ),
+                    StringOrNumberRange::Int(n) => format!(
+                        "{attribute} >= {start} AND {attribute} <= {stop}",
+                        attribute = attribute,
+                        start = n.start(),
+                        stop = n.end()
+                    ),
+                };
```
Comment (ChristianBeilschmidt) [inline operators/src/source/ogr_source/mod.rs:563]: I like this abstraction. Could we optimize `=` if `start == stop`?

**Review by ChristianBeilschmidt — CHANGES_REQUESTED** [review]

File: operators/src/source/ogr_source/mod.rs (hunk 1 above)
Comment (michaelmattig) [reply operators/src/source/ogr_source/mod.rs:581]: 62908f874ce3f398f188593bac5bf66d3fc6bd46

**Review by ChristianBeilschmidt — APPROVED** [review]

**Comment by ChristianBeilschmidt** [comment]
bors merge

**Pull request merged**